src/thunderbots/software/util/math_functions.h

```diff
@@ -0,0 +1,18 @@
+//
+// Created by roark on 28/03/19.
+//
+
+#pragma once
+
+namespace Util
+{
+    /**
+     * Linearly maps an input value to an output value in the range [0,1]
+     *
+     * @param value The input value
+     * @param offset The value by which to offset the input value
```

Comment:
Maybe also add "The input value at which the linear function will return 0.5". Just to help people understand if they give an offset of 0, the linear function is "centered" at 0, and if they input 1, the function is "centered" at 1.